src/components/header/Header.js

```diff
@@ -0,0 +1,30 @@
+import { Menu } from "@mui/icons-material";
+import NotificationsNoneOutlinedIcon from "@mui/icons-material/NotificationsNoneOutlined";
+import AccountCircleSharpIcon from "@mui/icons-material/AccountCircleSharp";
+import SearchIcon from "@mui/icons-material/Search";
```

Comment:
you can have a separate file for mui icons and export from there, in that way you will be importing your icons only from a single source.